data/json/items/ranged.json

```diff
@@ -4213,6 +4213,38 @@
     "valid_mod_locations": [[ "accessories", 4 ],[ "grip", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
   },
   {
+    "id": "m41a_pulse_rifle",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "teal",
+    "name": "M41A pulse rifle",
+    "description": "A powerful electrolaser  developed by the R&D outfit \"Aerial Industries\".  Especially effective against electronic targets.",
+    "price": 1600000,
+    "material": ["steel", "plastic"],
+    "ammo_effects": ["DRAW_LASER_BEAM", "EMP", "BEANBAG"],
```

Comment:
Why add a new tag rather than just using LASER? It deals quite a bit of damage somehow;

That BEANBAG here could make it a fair bit too strong, though. Test it out on a hulk and you'll see what I mean.